### DISCUSSION TITLE
Usage not only `Number` as a layer `x`/`y` argument

### DISCUSSION THREAD

**Discussion by AndreiKingsley** [JetBrains/lets-plot-kotlin#190] [General]
At the moment, if I want to make a constant x in some layer, that value has to be `Number`. Why can't I use `String` or datetime type?

**Comment by alshan** [comment]
Constants accept values of a type matching the type of correspondent aesthetic. 
In case of positional aesthetics this is number.

If you want to use a value belonging to your data domain, you can try to use it in mapping: 
`geom_abc() { x=["my value"]}`
